packages/atlas/src/views/studio/YppDashboard/YppDashboard.tsx

```diff
@@ -76,6 +79,23 @@ export const YppDashboard: FC = () => {
             </TierWrapper>
           )}
         </Header>
+        {currentChannel?.isSuspended && (
+          <SuspendedInfoWrapper>
+            <SuspendedTitle>
+              <SvgAlertsError24 />
+              <Text variant="h300" as="p">
+                This channel has been suspended in the YouTube Partner Program
+              </Text>
+            </SuspendedTitle>
+            <Text variant="t200" as="p" color="colorCoreNeutral200">
+              To learn more about the reason behind the suspension, please reach out on the{' '}
+              <Button variant="primary" _textOnly to={atlasConfig.features.ypp.suspendedSupportLink ?? ''}>
+                {atlasConfig.features.ypp.suspendedLinkText ?? 'link destination'}
+              </Button>
+              . You won't be rewarded for doing tasks during the time this channel is suspended.
+            </Text>
+          </SuspendedInfoWrapper>
+        )}
```

Comment:
We already have a component for this. It's called `Banner. We should use it instead.